astroquery/mast/core.py

```diff
@@ -110,25 +147,17 @@ def _mashup_json_to_table(json_obj, col_config=None):
         # Removing "_selected_" column
         if col == "_selected_":
             continue
-
+        
         # reading the colum config if given
         ignore_value = None
         if col_config:
             col_props = col_config.get(col, {})
             ignore_value = col_props.get("ignoreValue", None)
 
-        # making type adjustments
-        if atype == "string":
-            atype = "str"
-            ignore_value = "" if (ignore_value is None) else ignore_value
-        if atype == "boolean":
-            atype = "bool"
-        if atype == "int":  # int arrays do not admit Non/nan vals
-            atype = np.int64
-            ignore_value = -999 if (ignore_value is None) else ignore_value
-        if atype == "date":
-            atype = "str"
-            ignore_value = "" if (ignore_value is None) else ignore_value
+        # regularlizing the type
+        reg_type = _parse_type(atype)
```

Comment:
just in case might want to lower case this value

Reply:
`_parse_type` handles the lowercasing.